src/main/java/restfulbooking/Booking.java

```diff
@@ -0,0 +1,41 @@
+package restfulbooking;
+
+import com.google.gson.*;
+
+import java.time.LocalDate;
+
+public class Booking
+{
+	static Gson gson = new GsonBuilder()
+			.setPrettyPrinting().setDateFormat("yyyy-MM-Dd")
+			.registerTypeAdapter(LocalDate.class, new LocalDateGsonAdapter())
```

Comment:
This is just for converting back and forth from JSON to object type.  There is some trickery for converting dates ... see more details here:https://stackoverflow.com/questions/39192945/serialize-java-8-localdate-as-yyyy-mm-dd-with-gson